packages/http-client-java/emitter/src/options.ts

```diff
@@ -0,0 +1,26 @@
+import { JSONSchemaType } from "@typespec/compiler";
+
+// typespec-java has another "options.ts" file, with same "export".
+// If add/remove "export" here, please also check typespec-java in autorest.java repository.
+
+export const LIB_NAME = "@typespec/http-client-java";
+
+export interface DevOptions {
+  "generate-code-model"?: boolean;
+  debug?: boolean;
+  loglevel?: "off" | "debug" | "info" | "warn" | "error";
+  "java-temp-dir"?: string; // working directory for java codegen, e.g. transformed code-model file
+}
+
+export interface EmitterOptions {
+  "dev-options"?: DevOptions;
+}
+
+export const EmitterOptionsSchema: JSONSchemaType<EmitterOptions> = {
+  type: "object",
+  additionalProperties: true,
+  properties: {
+    "dev-options": { type: "object", additionalProperties: true, nullable: true },
```

Comment:
seems like dev-options should have a nested schema no?

Reply:
no, I don't think we really considered nested schema much. We'll have to see how to render the docs there

Reply:
does it crash or just not show what they look like? 
either way can you file an issue

Reply:
issue [here](https://github.com/microsoft/typespec/issues/6478).